beacon-chain/rpc/validator/status.go

```diff
@@ -118,14 +121,11 @@ func (vs *Server) validatorStatus(ctx context.Context, pubKey []byte, headState
 		}
 	}
 	// Our position in the activation queue is the above index - our validator index.
-	queuePosition = idx - lastActivatedValidatorIdx
-	return &pb.ValidatorStatusResponse{
-		Status:                    vStatus,
-		Eth1DepositBlockNumber:    eth1BlockNumBigInt.Uint64(),
-		PositionInActivationQueue: queuePosition,
-		DepositInclusionSlot:      depositBlockSlot,
-		ActivationEpoch:           headState.Validators[idx].ActivationEpoch,
+	if lastActivatedValidatorIdx > idx {
```

Comment:
this line doesn't make sense , how can the last activated validator index be greater than the index for a non-activated validator ?

Reply:
I agree, i'm just being defensive here to prevent any panic

Reply:
oh but its not possible

Reply:
Wait, it is technically possible. I'm trying to prevent a panic so i'll leave this code here even if it feels a bit strange.

Reply:
I guess it will probably never happen then ? Sounds good to leave it for now

Reply:
It _should_ never happen 

/shrug

Reply:
also it's not to prevent panic, but an underflow